.vscode/settings.json

```diff
@@ -14,7 +14,8 @@
         "${workspaceFolder}/src/lib/**",
         "${workspaceFolder}/src/system/**",
         "${workspaceFolder}/third_party/nlassert/repo/include/**",
-        "${workspaceFolder}/third_party/nlio/repo/include/**"
+        "${workspaceFolder}/third_party/nlio/repo/include/**",
+        "${NRF5_SDK_ROOT}/**"
```

Comment:
I don't think anything in tne NRF_SDK_ROOT should be a header in our projects, though...

Reply:
You can do this instead:
- Run the 'Update compilation database' task
- Set the build configuration to nordic examples

This will extract include paths using the build files, so will be fully accurate and include SDK paths. In fact this whole list could be removed.